instrumentation/apache-httpasyncclient-4.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachehttpasyncclient/ApacheHttpAsyncClientHttpAttributesExtractor.java

```diff
@@ -23,23 +23,6 @@ protected String url(ApacheHttpClientRequest request) {
     return request.getUrl();
   }
 
-  @Override
-  protected String target(ApacheHttpClientRequest request) {
-    return request.getTarget();
```

Comment:
Should we remove these method from `ApacheHttpClientRequest` too? They should all be unused now

Reply:
(also applies to other Apache HTTP instrumentations)